src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -284,17 +284,17 @@ private void ConfigureApi(IApplicationBuilder app)
 
             app.UseAuthentication();
 
-            //if (HostingEnvironment.IsDevelopment() && !Program.RunningInServiceFabric())
-            //{
-            //    // Redirect api requests to prod when running locally outside of service fabric
-            //    // This is for the `ng serve` local debugging case for the website
-            //    app.MapWhen(
-            //        ctx => IsGet(ctx) && ctx.Request.Path.StartsWithSegments("/api") && ctx.Request.Path != "/api/swagger.json",
-            //        a =>
-            //        {
-            //            a.Run(ApiRedirectHandler);
-            //        });
-            //}
+            if (HostingEnvironment.IsDevelopment() && !Program.RunningInServiceFabric())
```

Comment:
Should this change be retained?

Reply:
I commented this block of code only to do local tests against my local BAR. I committed the code by accident.